src/app/clusters/door-lock-server/door-lock-server.cpp

```diff
@@ -1659,8 +1659,7 @@ EmberAfStatus DoorLockServer::createUser(chip::EndpointId endpointId, chip::Fabr
     {
         emberAfDoorLockClusterPrintln("[createUser] Unable to overwrite existing user [endpointId=%d,userIndex=%d]", endpointId,
                                       userIndex);
-        // TODO: Add cluster-specific failure of DlStatus::kOccupied
-        return EMBER_ZCL_STATUS_FAILURE;
+        return static_cast<EmberAfStatus>(DlStatus::kOccupied);
```

Comment:
This is not right.  This is going to send this value as a _global_ status.  But this is a cluster-specific status, no?

It needs to be encoded differently than a global status on the wire, and hence we need to tell the CommandHandler which one we are sending.

Reply:
Got it. Is there any example on how to return the cluster-specific code?